### PR TITLE
Change key type into key family

### LINKED ISSUE #4376: KeyIdentity.type should be called family
Let's refactor this to `family: KeyFamily` in some other issue? It would make a good name I think, and it's already called that in some code.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-browser/pull/4372#discussion_r845113562_

### PR DESCRIPTION
**Pull request by IvanPizhenko**
This PR changes "key type" into "key family".

close #4376

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: extension/chrome/settings/setup.htm [hunk 1]
```diff
@@ -147,7 +147,7 @@ <h1>Set Up FlowCrypt</h1>
 
       <div class="line left">
         <label>
-          Encryption key type:
+          Encryption key family:
```
Comment (tomholub) [inline extension/chrome/settings/setup.htm:150]: Here the "key type" referred to algorithms, not openpgp vs x509

Comment (IvanPizhenko) [reply extension/chrome/settings/setup.htm]: ok

**Comment by IvanPizhenko** [comment]
@tomholub @rrrooommmaaa Ready for review

**Review by rrrooommmaaa — CHANGES_REQUESTED** [review]
we actually need to rename all the related internal variables too

**Comment by IvanPizhenko** [comment]
@rrrooommmaaa Renamed that variable, please recheck

**Review by rrrooommmaaa — APPROVED** [review]

**Pull request merged**